python/python/tests/test_async_client.py

```diff
@@ -7925,6 +7925,7 @@ async def test_sscan(self, redis_client: GlideClusterClient):
                 set(next_result[result_collection_index])
             )
             result_values.update(next_result[result_collection_index])
+            result = next_result
```

Comment:
How it worked before?

Reply:
Unfortunately, it only compared the next result set to the initial result set. I made it so the we check that every iteration has a different set from the previous iteration.